packages/clay-form/src/Form.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+
+import classNames from 'classnames';
+import ClayIcon from '@clayui/icon';
+
+import Checkbox from './Checkbox';
+import Input from './Input';
+import Radio from './Radio';
+import RadioGroup from './RadioGroup';
+
+const ClayForm: React.FunctionComponent<
+	React.HTMLAttributes<HTMLDivElement>
+> & {
+	Checkbox: typeof Checkbox;
+	FeedbackGroup: typeof FeedbackGroup;
```

Comment:
Normally I try to follow the order of declaration than read logic, I prefer to see the declared methods first and then consume, but it's just my opinion.

Reply:
Hmm, not sure what you mean by this. Can you share a code example?

Reply:
Oh sorry for that, What I mean is: I almost always try to prioritize the organization of methods, variables... in order of declaration ie I declare first before using. Example https://github.com/liferay/clay/blob/master/packages/clay-data-provider/src/useResource.ts

I think this eslint rule should explain better than me https://eslint.org/docs/rules/no-use-before-define 😅

Reply:
I [asked a related question on Twitter](https://twitter.com/wincent/status/1153033906922360835) and was surprised by the variation in the responses (not so much that there were people with different opinions, but rather that there were so many people who thought their opinions were the only ones that were sane — I guess I should have expected that 😂).

Reply:
Ohh my god! I think we can come to a consensus here haha